### DISCUSSION TITLE
Add metadata unicode support while /np in chat

### DISCUSSION THREAD

**Discussion by troke12** [ppy/osu#13156] [Ideas]
**Describe the new feature:** 

if the metadata turn on, they will change to preferred metadata, i would suggest to add support in chat too

**Proposal designs of the feature:** 

![img](https://cdn.discordapp.com/attachments/408950289962369025/756330891550130196/unknown.png)

**Comment by peppy** [comment]
Do you mean for the person sending the chat? Because that's the only way it can possibly work.

**Comment by troke12** [comment]
Yes
